css/style.css

```diff
@@ -2,10 +2,146 @@
 * {
   margin: 0;
   padding: 0;
+
 }
 
 header {
   height: 700px;
   background: url("../images/header_bg.jpg") no-repeat center top;
   background-size: cover;
+  height: 400px;
+}
+
+body{
+	font-family:'Open Sans', sans-serif;
+	width: 720px;
+	border-left: 75px solid #C9D3D6;
+	border-right: 75px solid #C9D3D6;
+	margin-right: 0;
+}
+
+.head{
+	font-weight: extra bold;
+	color: white;
+	padding-left: 50px;
+	padding-top: 25px;
+	font-size: 24px;
+}
+
+.int{
+	font-weight: 400 semibold;
+	font-size: 24px;
+	color: white;
+	padding: 85px;
+	font-size: 24px;
+}
+
+#benefit{
+	background-color: white;
+	padding: 25px 100px 25px 50px;
+
+}
+
+.ftw{
+	width: 375px;
+	font-size: 13px;
+	font-weight: light;
+}
+
+#quote{
+	background-image: url("../images/testimonial_bg.jpg");
+	color: white;
+	text-align: center;
+
+}
+
+.button{
+	background-color: #f9e42e;
+	margin: 0 auto 0 auto;
+	width: 125px;
+	text-align: center;
+	border-radius: 5px;
+	padding: 5px 15px 5px 15px;
+	font-size: 14px;
+	font-weight: extra bold;
```

Comment:
Extra bold is not a font-weight keyword. Also, you are better off using sizes (400,600,700,800).